api/index.js

```diff
@@ -53,9 +53,61 @@ async function fetchStats(username) {
 
   const user = res.data.data.user;
 
+  // FETCH ALL COMMITS
+  let commitPromises = user.contributionsCollection.contributionYears.map(
+    async (year) => {
+      // Don't fetch contributions older than 2008 (Perf optimization)
+      if (year < 2008) {
+        return {
+          user: { contributionsCollection: { totalCommitContributions: 0 } },
+        };
+      }
+
+      const currentDate = new Date();
+      currentDate.setFullYear(year, 0, 0);
+
+      const res = await axios({
+        url: "https://api.github.com/graphql",
+        method: "post",
+        headers: {
+          Authorization: `bearer ${process.env.GITHUB_TOKEN}`,
+        },
+        data: {
+          query: `
+          query userInfo($login: String!, $from: DateTime!) {
+            user(login: $login) {
+              contributionsCollection(from: $from) {
+                totalCommitContributions
+              }
+            }
+          }
+        `,
+          variables: {
+            login: username,
+            from: currentDate.toISOString(),
+          },
+        },
+      });
+
+      console.log(
```

Comment:
rm log?

Reply:
Hmmm yeah probably i'll remove... that although it's fine too helps me debug the serverless function with vercel logs

Reply:
Then you should probably only do it in case of exceptions which need debuggin, else you're just logging normal queries unless that's what you're intending